python/core/opening_utils.py

```
@@ -671,7 +671,68 @@ def normalize_opening_name(opening: str) -> str:
+    lines = pgn.split('\n')
+    moves_started = False
+    first_move = None
+
+    for line in lines:
+        line = line.strip()
+        # Skip headers and empty lines
+        if not line or line.startswith('['):
+            continue
+        # Found the moves section
+        if line and not line.startswith('['):
+            moves_started = True
```

Ruff (0.14.2)

Local variable `moves_started` is assigned to but never used

Remove assignment to unused variable `moves_started`

(F841)